pages/providers/email.html

```diff
@@ -7,13 +7,12 @@
 
 <div class="card border-danger">
   <div class="card-header text-danger"><i class="fas fa-exclamation-circle fa-fw"></i> Warning</div>
-  <div class="card-body">
-      <p class="card-text text-danger">When using end-to-end encryption technology like <a href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy">PGP</a>, email will still have <a href="https://en.wikipedia.org/wiki/Email#Message_header">some metadata</a> that is not encrypted end-to-end.</p>
-      <p><a href="https://en.wikipedia.org/wiki/Opportunistic_TLS">Opportunistic TLS</a> is used protect email transmission between servers however, both your email provider and the recipient(s) will be able to see the metadata.<p>
-      <p class="card-text text-danger">PGP also does not support <a href="https://en.wikipedia.org/wiki/Forward_secrecy">Forward secrecy</a>. If either your private key, or the recipient's is ever stolen <strong>all</strong> previous messages encrypted with it will be exposed.</p>
-      <p>To protect against private key theft it is recommended that they be stored on a <a href="https://en.wikipedia.org/wiki/OpenPGP_card">OpenPGP card</a>. When a OpenPGP card is used, the message is decrypted on the card and sent back to the device you wish to view it on. The private key <strong>never leaves the card</strong>, so if you view a message on a compromised device, only that particular message may be stolen. Your private key will remain safe, and so will any encrypted material you didn't view on the compromised device.</p>
-      <p>Alternatively consider moving prolonged conversation to a medium that does support <a href="https://en.wikipedia.org/wiki/Forward_secrecy">Forward secrecy</a>.</p>
-      <a href="/software/real-time-communication/" class="btn btn-outline-secondary">Recommended Instant Messengers</a></div>
+    <div class="card-body">
+      <p class="card-text text-danger">When using end-to-end encryption technology like <a href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy">PGP</a>, email will still have some metadata that is not encrypted in the header of the email. More information about <a href="/providers/email/#info">email metadata</a>.</p>
```

Comment:
```suggestion
      <p class="card-text text-danger">When using end-to-end encryption technology like <a href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy">PGP</a>, email will still have some metadata that is not encrypted in the header of the email. <a href="/providers/email/#info">Read more about email metadata.</a></p>
```

LGTM, but I would probably make this link a call-to-action.